### ISSUE TITLE
Theme: Patterns does not work in shinyserver (shinyapps.io)

### ISSUE THREAD

**Issue by debsush** [datastorm-open/rAmCharts#47]
Hi,

The theme patterns does not work in shinyserver (shinyapps.io).  

Also was curious to see if there is a theme called "black" available.  I could see js file inside the package but when I tried theme="black", I got an error so not sure

PS:  I am using stockchart

**Comment by Djaiff** [comment]
Hi,

Apparently, everything works in rAmCharts. It seems like it's a new improvement of amcharts.js which isn't finalized. You can see it by changing the theme in the simple example at https://www.amcharts.com/demos/column-with-rotated-series/

Best,
Jeff

**Comment by debsush** [comment]
@Djaiff 

Please see my webapp in action (hope you like the dev version).  If you draw a chart and then go to settings (Cog) and change the theme to "Patterns" [4th button], you will find that stripes do not appear when the app is deployed in shinyapps.io  

The same patterns appear perfectly well when run locally.  Same issue happens when you change the language.  It runs fine locally but not over shinyapps.io 
PS: Shinyapps.io is in https and bars all calls to http.  

https://dmarket.shinyapps.io/TerminalBeta (made with v 1.1.3)

Thank you
SD

**Comment by Djaiff** [comment]
Could you try to set this as first level parameter of your chart ?
`path = "http://www.amcharts.com/lib/3"`

**Comment by debsush** [comment]
Jeff:

Shinyapps.io will throw error if it is http.  Can I use https://

**Comment by debsush** [comment]
I guess I can use https.  I will try

**Comment by debsush** [comment]
I did what you suggested but no luck yet.  I pushed the commit in shinyapps.io but neither the languages nor the pattern theme works.

```
chart <<- pipeR::pipeline(
                  amStockChart(startDuration = 0,
                               path = 'http://www.amcharts.com/lib/3',
                               .............
```

In fact for languages, if you open the browser console when running my app, you will see the following error
Mixed Content: The page at 'https://dmarket.shinyapps.io/TerminalBeta/' was loaded over HTTPS, but requested an insecure script 'http://www.amcharts.com/lib/3/lang/fr.js'. This request has been blocked; the content must be served over HTTPS.
